samcli/commands/deploy/guided_context.py

```diff
@@ -169,9 +167,14 @@ def guided_prompts(self, parameter_override_keys):
                 type=click.STRING,
             )
 
+        click.echo("\n\tLooking for resources needed for deployment:")
         s3_bucket = manage_stack(profile=self.profile, region=region)
-        click.echo(f"\n\t\tManaged S3 bucket: {s3_bucket}")
-        click.echo("\t\tA different default S3 bucket can be set in samconfig.toml")
+        click.echo(f"\t Managed S3 bucket: {s3_bucket}")
+        click.echo("\t A different default S3 bucket can be set in samconfig.toml")
```

Comment:
Applies to most prompts/outputs, there's an extra space on top of the tab:
```suggestion
        click.echo(f"\tManaged S3 bucket: {s3_bucket}")
        click.echo("\tA different default S3 bucket can be set in samconfig.toml")
```

Reply:
The spaces were added on purpose as required by UX.